src/model/entity.model.ts

```diff
@@ -16,6 +16,8 @@ export const PrincipalAddressKeys: string[] = ["principalAddressPropertyNameNumb
 export const ServiceAddressKey = "service_address";
 export const ServiceAddressKeys: string[] = ["serviceAddressPropertyNameNumber", "serviceAddressLine1", "serviceAddressLine2", "serviceAddressTown", "serviceAddressCounty", "serviceAddressCountry", "serviceAddressPostcode"];
 
+export const HasSameAddressKey = "is_service_address_same_as_principal_address";
```

Comment:
HasSameAddressKey i would start with a lower case 'h' hasSameAddressKey

Reply:
actually just realised this is inline with other key names in module, ignore my comment